lib/handbook/addon/docs/components/schema-block-group-renderer/controller.ts

```diff
@@ -0,0 +1,155 @@
+            index: 7,
+        },
+        {
+            blockType: 'short-text-input',
+            registrationResponseKey: 'page-one_short-text',
+            schemaBlockGroupKey: 'q2',
+            index: 8,
+        },
+        {
+            blockType: 'question-label',
+            displayText: 'What is the difference between a swamp and a marsh?',
+            schemaBlockGroupKey: 'q3',
+            index: 9,
+        },
+        {
+            blockType: 'long-text-input',
+            registrationResponseKey: 'page-one_long-text',
+            schemaBlockGroupKey: 'q3',
+            index: 10,
+        },
+        {
+            blockType: 'question-label',
+            displayText: 'I never understood all the hate for:',
+            schemaBlockGroupKey: 'q4',
+            index: 11,
+        },
+        {
+            blockType: 'multi-select-input',
+            registrationResponseKey: 'page-one_multi-select',
+            schemaBlockGroupKey: 'q4',
+            index: 12,
+        },
+        {
+            blockType: 'select-input-option',
+            displayText: 'Nickelback',
+            schemaBlockGroupKey: 'q4',
+            index: 13,
+        },
+        {
+            blockType: 'select-input-option',
+            displayText: 'Crocs',
+            schemaBlockGroupKey: 'q4',
+            index: 14,
+        },
+        {
+            blockType: 'select-other-option',
+            displayText: 'Other:',
+            schemaBlockGroupKey: 'q4',
+            index: 15,
+        },
+        {
+            blockType: 'question-label',
+            displayText: 'If I had a super power it would be:',
+            schemaBlockGroupKey: 'q6',
+            index: 17,
+        },
+        {
+            blockType: 'single-select-input',
+            registrationResponseKey: 'page-one_single-select-two',
+            schemaBlockGroupKey: 'q6',
+            index: 18,
+        },
+        {
+            blockType: 'select-input-option',
+            displayText: 'Always be on the proper beat while doing the macarena',
+            schemaBlockGroupKey: 'q6',
+            index: 19,
+        },
+        {
+            blockType: 'select-input-option',
+            displayText: 'Remember who was in NSync and who was in Backstreet Boys',
+            schemaBlockGroupKey: 'q6',
+            index: 20,
+        },
+        {
+            blockType: 'select-other-option',
+            displayText: 'Other',
+            schemaBlockGroupKey: 'q6',
+            index: 21,
+        },
+        {
+            blockType: 'question-label',
+            displayText: 'Contributors:',
+            schemaBlockGroupKey: 'q5',
+            index: 22,
+        },
+        {
+            blockType: 'contributors-input',
+            registrationResponseKey: 'page-one_contributors-input',
+            schemaBlockGroupKey: 'q5',
+            index: 23,
+        },
+    ];
+
+    schemaBlockGroup = getSchemaBlockGroups(this.schemaBlocks);
```

Comment:
```suggestion
    schemaBlockGroups = getSchemaBlockGroups(this.schemaBlocks);
```
Should be plural. Please don't hate me.